docs/csharp/language-reference/keywords/operator.md

```diff
@@ -12,9 +13,19 @@ ms.assetid: 59218cce-e90e-42f6-a6bb-30300981b86a
 
 Use the `operator` keyword to overload a built-in operator or to provide a user-defined conversion in a class or struct declaration.
 
+To overload an operator on a custom class or struct, you create an operator declaration in the corresponding type. The operator declaration that overloads a built-in C# operator must satisfy the following rules:
+
+- It includes both a `public` and a `static` modifier.
+- It includes `operator X` where `X` is the name or symbol of the operator being overloaded.
+- Unary operators have one parameter, and binary operators have two parameters. In each case, at least one parameter must be the same type as the class or struct that declares the operator.
+
+For the information about how to define conversion operators, see the [explicit](explicit.md) and [implicit](implicit.md) keyword articles.
+
+For the overview of the C# operators that can be overloaded, see the [Overloadable operators](../../programming-guide/statements-expressions-operators/overloadable-operators.md) article.
```

Comment:
- the overview --> an overview
- I think this is more effective if it's moved into the second bulleted list item. Though it's also fine here.